src/zjs_net_config.c

```diff
@@ -1,6 +1,49 @@
-// Copyright (c) 2016-2017, Intel Corporation.
+// Copyright (c) 2017, Intel Corporation.
+
+#include <net/net_context.h>
+#include <net/nbuf.h>
+#include <net/net_if.h>
+#if defined(CONFIG_NET_L2_BLUETOOTH)
+#include <bluetooth/bluetooth.h>
+#include <gatt/ipss.h>
+#endif
+#include "zjs_util.h"
+#include "zjs_net_config.h"
+
+static uint8_t net_enabled = 0;
+#if defined(CONFIG_NET_L2_BLUETOOTH)
+static uint8_t ble_enabled = 0;
+#endif
 
-#include "zjs_ocf_ble.h"
+void zjs_net_config(void)
+{
+#if defined(CONFIG_NET_L2_BLUETOOTH)
+    if (!ble_enabled) {
+        zjs_init_ble_address();
+        if (bt_enable(NULL)) {
```

Comment:
what is the return value of bt_enable here? if it not boolean maybe check if != error?